app/res/values-hi/strings.xml

```diff
@@ -0,0 +1,609 @@
+    <string name="personalid_configuration_process_failed_unexpected_error">एक अप्रत्याशित त्रुटि के कारण डिवाइस पात्रता स्थापित करने में असमर्थ। यदि समस्या बनी रहती है तो कृपया ग्राहक सहायता से संपर्क करें</string>
+    <string name="personalid_configuration_process_failed_security_subtitle">%s सुरक्षा सुविधा की अनुपलब्धता के कारण आपका डिवाइस इस समय PersonalID के लिए साइन अप करने के योग्य नहीं है। कृपया किसी भिन्न डिवाइस पर पुनः प्रयास करें।</string>
+    <string name="personalid_photo_capture_title">स्वागत है %1$s</string>
+    <string name="personalid_photo_capture_subtitle">अपना खाता सेटअप पूरा करने के लिए कृपया अपनी एक तस्वीर लें</string>
+    <string name="personalid_photo_capture_take_photo_button">तस्वीर लें</string>
+    <string name="personalid_photo_capture_save_photo_button">तस्वीर सहेजें</string>
+    <string name="personalid_photo_upload_failure">"तस्वीर सहेजने में विफल। कृपया अपना इंटरनेट जांचें और पुनः प्रयास करें। "</string>
+    <string name="personalid_this_is_not_me"><u>यह मैं नहीं हूँ</u></string>
+    <string name="personalid_welcome_back_msg">वापस स्वागत है %s</string>
+    <string name="personalid_recovery_failed_title">पुनर्प्राप्ति विफल</string>
+    <string name="personalid_recovery_lockout_title">खाता लॉक</string>
+    <string name="personalid_recovery_failed_message">ऐसा लगता है कि आप अपना बैकअप कोड भूल गए हैं, हम आपके लिए एक नया खाता बना रहे हैं</string>
+    <string name="personalid_recovery_lockout_message">आपने कई बार गलत बैकअप कोड दर्ज किया है और आपका खाता लॉक कर दिया गया है।</string>
+    <string name="personalid_wrong_backup_message">आपने गलत बैकअप कोड दर्ज किया, कृपया पुनः प्रयास करें। %d और गलत प्रयासों के बाद आपको एक नया खाता बनाना होगा।</string>
+    <string name="personalid_camera_permission_title">कैमरे के लिए अनुमति</string>
+    <string name="personalid_camera_permission_msg">एक तस्वीर लेने के लिए, CommCare को आपके डिवाइस कैमरे का उपयोग करने की अनुमति चाहिए।</string>
+    <string name="personalid_capture_photo">तस्वीर कैप्चर करें</string>
+    <string name="personalid_incorrect_otp">दर्ज किया गया OTP गलत है। कृपया पुनः प्रयास करें।</string>
+    <string name="personalid_too_many_otp_attempts">बहुत अधिक प्रयास। कृपया बाद में पुनः प्रयास करें।</string>
+    <string name="personalid_otp_missing_activity">कुछ गलत हो गया</string>
+    <string name="personalid_otp_verification_failed_generic">OTP सत्यापन विफल:</string>
+    <string name="personalid_otp_verification_failed">सत्यापन विफल और कृपया पुनः प्रयास करें</string>
+    <string name="personalid_location_permission_error">स्थान अनुमति अस्वीकृत</string>
+    <string name="personalid_location_service_error">स्थान सेवा उपलब्ध नहीं है। कृपया इसे सक्षम करें</string>
+    <string name="personalid_grant_location_permission">स्थान अनुमति दें</string>
+    <string name="personalid_grant_location_service">स्थान सेवा सक्षम करें</string>
+    <string name="my_credentials">मेरे क्रेडेंशियल</string>
+    <string name="credential">क्रेडेंशियल</string>
+</resources>
```

Comment:
Curious why the # of lines here are different than https://github.com/dimagi/commcare-android/blob/master/app/res/values-fr/strings.xml. The string translations in the 2 files should mimick each other one to one.

Reply:
yes think it's safer to mimick french to be consistent with other languages.